client/src/graphics/voxels/mod.rs

```diff
@@ -142,7 +142,7 @@ impl Voxels {
         for &(node, ref node_transform) in &nodes {
             let node_to_view = local_to_view * node_transform;
             let origin = node_to_view * math::origin();
-            if !frustum_planes.contain(&origin, dodeca::BOUNDING_SPHERE_RADIUS) {
+            if !frustum_planes.contain(&origin, BOUNDING_SPHERE_RADIUS) {
```

Comment:
Please leave this qualified as `dodeca::BOUNDING_SPHERE_RADIUS`, as it was originally, here and in the other occurrences. That form is intentional, as it makes it clear what bounding sphere we're talking about.